### PR TITLE
Add Host Installation Logs download action

### PR DESCRIPTION
**Pull request by mareklibra**
Depends on:
- [x] #171 
- [x] Use pre-signed URLs
- [x] remove stub for cluster logs (backend not ready)

### CODE REVIEW

File: src/config/constants.ts [hunk 1]
```diff
@@ -12,17 +12,22 @@ type OpenshiftVersionOptionType = {
 };
 
 export const OPENSHIFT_VERSION_OPTIONS: OpenshiftVersionOptionType[] = [
-  // { label: 'OpenShift 4.4', value: '4.4' },
   { label: 'OpenShift 4.5', value: '4.5' },
   // { label: 'OpenShift 4.6', value: '4.6' },
 ];
+export const DEFAULT_OPENSHIFT_VERSION: OpenshiftVersionOptionType['value'] = '4.5';
```
Comment (jtomasek) [inline src/config/constants.ts:18]:
```suggestion
export const DEFAULT_OPENSHIFT_VERSION: OpenshiftVersionOptionType['value'] = OPENSHIFT_VERSION_OPTIONS[0].value;
```

File: src/components/clusterDetail/ClusterInstallationError.tsx [hunk 2]
```diff
@@ -23,9 +24,26 @@ const ClusterInstallationError: React.FC<ClusterInstallationErrorProps> = ({
...
+          >
+            Download Installation Logs
+          </AlertActionLink>
+          */}
+          <AlertActionLink
+            onClick={() => {
+              window.open(getBugzillaLink(cluster.openshiftVersion), '_blank');
+            }}
+          >
+            Create A New Bugzilla
```
Comment (jtomasek) [inline src/components/clusterDetail/ClusterInstallationError.tsx:44]:
```suggestion
            Report a bug
```

**Review by jtomasek — APPROVED** [review]

File: src/components/clusterDetail/ClusterInstallationError.tsx (hunk 2 above)
Comment (mareklibra) [reply src/components/clusterDetail/ClusterInstallationError.tsx:44]: per design but I would agree with this suggestion

File: src/config/constants.ts (hunk 1 above)
Comment (mareklibra) [reply src/config/constants.ts:18]: will push a follow-up

**Pull request merged**